modules/appeals_api/app/controllers/appeals_api/v1/decision_review/higher_level_reviews_controller.rb

```diff
@@ -1,22 +1,50 @@
 # frozen_string_literal: true
 
+require 'json_schema/json_api_missing_attribute'
+require 'appeals_api/form_schemas'
+require_dependency 'appeals_api/concerns/json_format_validation'
+
 class AppealsApi::V1::DecisionReview::HigherLevelReviewsController < AppealsApi::ApplicationController
+  include AppealsApi::JsonFormatValidation
+
   skip_before_action(:authenticate)
+  before_action :validate_json_format, if: -> { request.post? }
+  before_action :validate_json_schema, only: %i[create validate]
+
+  FORM_NUMBER = '200996'
 
   def show
     render_response(Appeals::Service.new.get_higher_level_review(params[:id]))
   end
 
   def create
-    render_response(Appeals::Service.new.create_higher_level_review(create_params))
+    render json: { data: { success: true } }
+  end
+
+  def validate
+    render json: validation_success
+  end
+
+  def schema
+    render json: { data: [AppealsApi::FormSchemas.new.schemas[self.class::FORM_NUMBER]] }
```

Comment:
maybe in the future we can think about convert the form schema stuff into a concern so schema files aren't being read all the time.

Reply:
or caching the schemas at the class level of  `AppealsApi::JsonFormatValidation`